newsletter-april-2021.rst

```diff
@@ -0,0 +1,116 @@
+.. post:: April 6, 2021
+   :tags: newsletter
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features from April.
+
+Read the Docs newsletter - April 2021
+=====================================
+
+This is the first of our monthly newsletters, in which we would like to
+openly share with you the most relevant updates of Read the Docs,
+offer a summary of what new features we shipped to our users
+during the previous month,
+and share what things we will be focusing on in the near future.
+
+Company highlights
+------------------
+
+-  We have a new colleague! `Juan Luis`_
+   will be working with us as Developer Advocate, with a focus on fulfilling
+   the goals of `the CZI grant we were
+   awarded <https://blog.readthedocs.com/czi-grant-announcement/>`_,
+   improve our public facing documentation,
+   and spread the word about our service.
+-  The average daily pageviews went up by 9.4 % from the previous month.
+-  We have `a new status page <http://status.readthedocs.com/>`_ so you
+   can see the availability history and current status of our infrastructure.
+
+New features
+------------
+
+-  There is a `new support form <https://docs.readthedocs.io/en/latest/support.html>`_
+   in case you need to send us a request related to your project or
+   account, like asking for more resources, change the project slug, and
+   so forth.
+-  You can now build your project using Python 3.9 by setting your build
+   image to ``testing``. Please `open an
+   issue <https://github.com/readthedocs/readthedocs.org/issues/new>`_
+   if you find any problems with it.
+-  Now `all newly created MkDocs projects use the latest version of mkdocs by
+   default <https://github.com/readthedocs/readthedocs.org/pull/7869>`_.
+   Existing projects can specify a specific version, but won't automatically
+   be upgraded.
+-  We expanded a few areas of our user documentation: we added `a brand
```

Comment:
We could mention here that the correct guide to contribute is now `install.html` instead of `standards.html` :smile:

See https://github.com/readthedocs/readthedocs.org/pull/7994

Reply:
On it

Reply:
This could perhaps use it's own section, noting Docker Compose being the new recommended method.

Reply:
Done!